packages/nuxt/src/vite/utils.ts

```diff
@@ -24,3 +24,55 @@ export function findDefaultSdkInitFile(type: 'server' | 'client'): string | unde
 
   return filePaths.find(filename => fs.existsSync(filename));
 }
+
+export const SENTRY_WRAPPED_ENTRY = '?sentry-query-wrapped-entry';
+export const SENTRY_FUNCTIONS_REEXPORT = '?sentry-query-functions-reexport=';
+export const QUERY_END_INDICATOR = 'SENTRY-QUERY-END';
+
+/**
+ * Strips a specific query part from a URL.
+ *
+ * Only exported for testing.
+ */
+export function stripQueryPart(url: string): string {
```

Comment:
nit: Maybe we can make this function name a bit more representative of what it does.

Reply:
Renamed it to `removeSentryQueryFromPath`